File: Services/Dto/SolutionPriceDto.cs
using System;

namespace Services.Dto

{
    public class SolutionPriceDto
    {
        public int Id { get; set; }
        public DateTime DateService { get; set; }
        public string Technician { get; set; }
        public decimal PriceService { get; set; }
        public string Technicalsolution { get; set; }
        public bool Authorized { get; set; }
        public bool Remote { get; set; }
    }
}

File: Views/src/app/_components/administrative/local/services/service-budget/dto/solution-price-dto.ts
import { EquipamentDto } from "../../items-services/dto/equipament-dto";

export class SolutionPriceDto {
  id: number;
  dateService: Date;
  technician: string;
  priceService: number;
  technicalSolution: string;
  authorized: boolean;
  remote: boolean;
  comment:string;
}
